dispatch-rules.js

```diff
@@ -366,6 +366,7 @@ rule = {
           }
           ?bestuurseenheid mu:uuid ?uuid
         }
+      }
```

Comment:
FYI; to find these, i removed the pre-filter and let all rules run.

Reply:
Oh nice catch ! Will remember the pre-filter trick. Thanks for the review